ydb/core/protos/kqp_physical.proto

```diff
@@ -507,4 +507,6 @@ message TKqpPhyQuery {
     bool HasUncommittedChangesRead = 9;
 
     string QueryDiagnostics = 10;
+
+    repeated TKqpTableInfo ViewsInfo = 11;
```

Comment:
`ViewsInfo` -> `ViewInfos`

1. the use of `ViewsInfo` is inconsistent with other intances of plural forms in that same message (and entire file)
2. compound nouns normally form the plural by adding -s to the last word of the compound